apps/vr-tests-react-components/src/stories/Button/Button.stories.tsx

```diff
@@ -21,7 +21,7 @@ export const DefaultHighContrast = getStoryVariant(Default, HIGH_CONTRAST);
 
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31479&commitId=c3b1ae38ead3170cb599606544e21f421243f898&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|4| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|3| Changed|

</details>


  <details><summary><b>Button Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Converged.As an anchor.default.chromium.png|308| Changed|
|Button Converged.As an anchor.pressed.chromium.png|300| Changed|
|Button Converged.As an anchor.hover.chromium.png|296| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>